libcloudforensics/providers/gcp/internal/compute.py

```diff
@@ -963,6 +994,81 @@ def Delete(
             self.FormatLogMessage(
                 'Could not find disk: {0:s}, skipping'.format(disk_name)))
 
+  def AssignExternalIp(self, net_if: str, ip_addr: Optional[str]) -> None:
+    """Assigns an external IP to an instance's network interface.
+
+    The instance must not have an IP assigned to the network interface when
```

Comment:
I think in the current scenario this wonn't happen since RemoveExternalIps removes all ips from all network interfaces and is just called before this in the call, but handling execute exceptions (maybe via ExecuteRequest) is for sure a good idea